elm-package-info-viewer/EVALUATION.md

```diff
@@ -0,0 +1,19 @@
+## Review Questions
+
+These are the questions that we should ask when reviewing a homework submission.
+
+* Is the app in one of the languages listed in the instructions? If not, did the applicant get special permission to use something else?
+* Does the submission include instructions for getting the assignment up and running locally.
+* Does the app meet each of the user story requirements set in the assignment
+* Is the code well organized?
+  * Is it split into multiple methods and views in an appropriate way?
+  * Does the code include comments when necessary to explain what's going on?
+* Does the code avoid repeating similar blocks of code for similar tasks?
+* Does the code use third party libraries and tools as appropriate?
+
+### Bonus Points
+
+These are not expected as a baseline, but their presence is a good thing.
+
+* Does the app handle loading and error states gracefully
+* Does the app handle missing data gracefully 
```

Comment:
Add a note about dynamic sizing.  Doesn't need to be stable on all screen sizes but if it doesn't scale at all that's a big red flag.